netci.groovy

```diff
@@ -209,6 +216,13 @@ def static addTriggers(def job, def isPR, def architecture, def os, def configur
             case 'forcerelocs':
             case 'jitstress1':
             case 'jitstress2':   
+            case 'js2_jsr1':
```

Comment:
I think that I can check whether the long names are fine before checkin. I will update the names if there is no problem.